.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 env:
   - HADOOP_VERSION=3.1.1
-script: "./gradlew test --stacktrace --info"
+script: "./gradlew test --stacktrace --info && ./gradlew ciPerformRelease"
```

Comment:
this will be called for every travis job - every PR/every branch

Reply:
should be fine. shipkit will not publish if it is a PR